src/drivetrain/drivetrain/drivetrain_node.py

```diff
@@ -173,10 +174,19 @@ def absolute_angle_reset(self):
         # self.front_left was chosen arbitrarily
         if self.front_left.current_absolute_angle is not None:
             print("Absolute Encoder angles reset")
-            self.front_left.reset(0)
-            self.front_right.reset(0)
-            self.back_left.reset(0)
-            self.back_right.reset(0)
+
+            front_left_future = self.cli_motor_get.call_async(MotorCommandGet.Request(type="position", can_id=self.FRONT_LEFT_TURN))
+            front_left_future.add_done_callback(lambda future: self.front_left.reset(future.result().data))
```

Comment:
Instead of passing in future.result().data (tachometer counts) directly, this needs to be converted to an angle, I think between 0 and 360 but not 100% sure on that last part yet.